HotSpot/View Controllers/MainContainerViewController.m

```diff
@@ -71,11 +75,40 @@ - (void)viewDidLoad {
     [self.searchResultTableView insertSubview:self.refreshControl atIndex:0];
     
     self.tableVC.initialLocation = self.mapVC.locationManager.location;
+    
+    // animations -- sets frame size to 0!
+    [self resetTableViewFrame];
+    
+    // I'm assuming that the people using HotSpot are Americans that speak english
+    self.speechRecognizer = [[SFSpeechRecognizer alloc] initWithLocale:[[NSLocale alloc] initWithLocaleIdentifier:@"en_US"]];
+   self.speechRecognizer.delegate = self;
+
+    // used to request authorization -- the different cases
+    [SFSpeechRecognizer requestAuthorization:^(SFSpeechRecognizerAuthorizationStatus status) {
+        switch (status) {
```

Comment:
if you're only logging the status, you don't need the switch at all... logging is for debugging purposes.